### ISSUE TITLE
JsonSerializer is case-sensitive so by default deserialized events are empty

### ISSUE THREAD

**Issue by DylanMunyard** [aws/aws-lambda-dotnet#183]
Hi - I noticed the Lambda event templates have a different casing to the event classes in this repo, so the lambda event is deserialised with empty values. If you see this commit by adding serialisation attributes the event is deserialised with data, https://github.com/DylanMunyard/aws-lambda-dotnet/commit/5fd23b4c7aae7af088b61935b6eef9a51484833e.

``` c#
[LambdaSerializer(typeof(Amazon.Lambda.Serialization.Json.JsonSerializer))]
public LexResponse Handler(LexEvent lexEvent)
```

``` json
{
  "userId": "John"
}
```

lexEvent.UserId is empty. Expected it to be John, making the change mentioned above deserialises as expected.

I'm very new to Lambda with C# so I definitely appreciate I'm probably doing something wrong.

**Comment by normj** [comment]
Thanks for letting us know, I'll take a look

**Comment by normj** [comment]
I'm confused what is going on here. You can see from our tests that we are testing with "userId" casing. 

Here is the sample event: 
https://github.com/aws/aws-lambda-dotnet/blob/b56b97d8591eabd4b92e89128f306ed0703b10c0/Libraries/test/EventsTests/lex-event.json#L4

And the corresponding test:
https://github.com/aws/aws-lambda-dotnet/blob/b56b97d8591eabd4b92e89128f306ed0703b10c0/Libraries/test/EventsTests/EventTests.cs#L442

Is it possible something in your code or imported dependencies are changing Newtonsoft.Json's default settings?

**Comment by DylanMunyard** [comment]
Hi - I went back to basics and created a brand new app and it's working - I can run the Lex EventTests tests and can see they pass - so I must have changed a dependency somehow, sorry but all good now.

**Comment by normj** [comment]
Cool, I also shipped version 1.0.1 of Amazon.Lambda.LexEvents that contained your pull request.
